src/help/help_impl.cpp

```diff
@@ -12,9 +12,12 @@
    See the COPYING file for more details.
 */
 
+#define GETTEXT_DOMAIN "wesnoth-help"
```

Comment:
I'm not sure. If you look at the current state of 1.14 specific text-domains are hacked in to work around the string freeze but that should not be applicable now. At some point these strings ought to be marked for translation - 'Time of Day Schedule' is not, in master, and I think that's due to this being missing.

Reply:
Yes, I agree with making the strings translatable, and with moving them to the correct textdomain by removing the temporary hack that was added during string freeze. What I'm saying is that adding this `#define` will move all _other_ strings in the file (some 20 uses of `_("foo")` throughout help_impl.cpp) to another textdomain. I'm hoping that won't require translators to re-translate all of them, that's all.

Reply:
It might be unavoidable. Don't know what domain - if any - this file was in before (your previous comment in #3064 suggests it's not in any).

Reply:
It was in the `"wesnoth"` domain, which is the default when no other is specified:

```
$ grep -l 'help_impl.cpp' po/*/*.pot
po/wesnoth/wesnoth.pot
```

Reply:
And I wonder if we can, like, just move all strings _with their translations_ from po/wesnoth/ to po/wesnoth-lib/.

Reply:
Ohhhh, I see now. Well, should it stay in `wesnoth` or should it go in `wesnoth-help` or `wesnoth-lib`? Maybe `wesnoth-help` is intended for the long prose paragraphs?

Reply:
I suppose keeping the file in `wesnoth` is the easiest solution, but honestly, I'm not sure what's the difference between that and `wesnoth-lib` is. (And what's more, I'm not sure why leave the file in `wesnoth` but keep pulling those strings from `wesnoth-lib`. It's not like they'd have different translations in `wesnoth` and `wesnoth-lib`...)

Reply:
I just remember that 'Time of Day Schedule' didn't work when not using `wesnoth-help` but never mind. It's back to `wesnoth` now.

Reply:
I think these _should_ be moved to `wesnoth-help`, since they're pretty obviously part of the help system. (As for `wesnoth` vs `wesnoth-lib`, I think it might make sense to actually merge those two. I don't think anyone knows what the difference between them is supposed to be.)